dashboard/api.py

```diff
@@ -492,7 +493,16 @@ def is_exam_schedulable(user, course):
     """
     Check if a course is ready to schedule an exam or not
     """
-    schedulable_exam_runs = ExamRun.get_currently_schedulable(course)
+    now = now_in_utc()
+    # pick current and future runs
+    filter_query = Q(  # current exam runs
+        date_first_schedulable__lte=now,
+        date_last_schedulable__gte=now
+    )
+    filter_query.add(Q(date_last_eligible__gte=now.date()), Q.OR)  # future exam runs
+    filter_query.add(Q(course=course), Q.AND)  # selected course
```

Comment:
nit: I would find it clearer if we started the query with the course filter and then add the date filters.

Reply:
Done